src/twisted/internet/unix.py

```diff
@@ -171,10 +171,22 @@ def doRead(self):
             else:
                 return main.CONNECTION_LOST
 
-        if ancillary:
-            ancillaryData = ancillary[0][2]
-            fdCount = len(ancillaryData) // 4
-            fds = struct.unpack('i'*fdCount, ancillaryData)
+        for cmsgLevel, cmsgType, cmsgData in ancillary:
+            if (cmsgLevel != socket.SOL_SOCKET or
+                cmsgType != sendmsg.SCM_RIGHTS):
+                log.msg(
```

Comment:
I'm not sure such policy would be beneficial in this case: if this PR uses `twisted.logger`, logging becomes inconsistent throughout the module; making use `twisted.logger` all around would bring lots of unrelated changes.

I'll go with (optional) not changing the logging approach and I've filed a ticket to convert this module's `twisted.python.log` to `twisted.logger`: https://twistedmatrix.com/trac/ticket/9009